### PR TITLE
[New] Add stretchy@1.1.0 w/ git auto-update

### LINKED ISSUE #11882: [Request] Add stretchy
**Library name:** stretchy
**Library description:** Form element autosizing plugin
**Git repository url:** https://github.com/LeaVerou/stretchy
**npm package name or url** (if there is one): https://www.npmjs.com/package/stretchy
**License (List them all if it's multiple):** MIT
**Official homepage:** https://leaverou.github.io/stretchy/
**Wanna say something? Leave message here:** Thanks

### PR DESCRIPTION
**Pull request by sashberd**
close #11882, cc @LeaVerou

Pull request for issue: #11882 
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/LeaVerou/stretchy/tree/master
 * Official website (optional, not the repository): https://leaverou.github.io/stretchy
 * NPM package url (optional): https://www.npmjs.com/package/stretchy
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 37
   - Count of stars: 1159
   - Count of forks: 79
 * NPM download stats (optional):
   - Downloads in the last day: 3
   - Downloads in the last week: 6
   - Downloads in the last month: 25

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x ] This lib was not found on cdnjs repo
 * [ x] No already exist / duplicated issue and PR
 * [x ] The lib has notable popularity
   * [ x] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 800 downloads stats per month on npm registry
 * [ x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ x] Has valid tags for each versions (for git auto-update)
 * [ ] Auto-update setup
 * [ ] Auto-update target/source is valid.
 * [ ] Auto-update filemap is correct.

# Git commit checklist
 * [x ] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x ] The parent of the commit(s) in the PR is not old than 3 days.
 * [ x] Pull request is sending from a non-master branch with meaningful name.
 * [x ] Separate unrelated changes into different commits.
 * [ x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [ x] Close corresponding issue in commit message
 * [ x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/stretchy/package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  "description": "Form element autosizing, the way it should be.",
+  "filename": "stretchy.min.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/LeaVerou/stretchy.git"
+  },
+  "author": "Lea Verou",
+  "license": "MIT",
+  "keywords": [
+    "element autosizing"
```
Comment (maruilian11) [inline ajax/libs/stretchy/package.json]: Could you please enrich and modify `keywords` with `"form"`, `"element"`, `"autosizing"`. Thanks. (We hope there are at least 3 keywords in package.json)

File: ajax/libs/stretchy/package.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/LeaVerou/stretchy.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "*.scss",
```
Comment (maruilian11) [inline ajax/libs/stretchy/package.json]: We don't host `.scss`. Please remove it.

File: ajax/libs/stretchy/package.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+          "stretchy*",
+          "style*"
```
Comment (maruilian11) [inline ajax/libs/stretchy/package.json]: I think `style*` are for `index.html`. I think we just need `"stretchy*"` here. Thanks.

**Comment by sashberd** [comment]
@sufuf3 the PR was updated

**Comment by sufuf3** [comment]
@maruilian11 @extend1994 please help review this PR.
Thanks.

**Pull request merged**